apps/taxes/src/modules/avatax/order-fully-refunded/avatax-order-fully-refunded-adapter.ts

```diff
@@ -0,0 +1,43 @@
+import { ExpectedError } from "../../../error";
+import { createLogger } from "../../../logger";
+import { OrderFullyRefundedPayload } from "../../../pages/api/webhooks/order-fully-refunded";
+import { WebhookAdapter } from "../../taxes/tax-webhook-adapter";
+import { AvataxClient } from "../avatax-client";
+import { AvataxConfig } from "../avatax-connection-schema";
+import { AvataxOrderRefundedPayloadTransformer } from "./avatax-order-fully-refunded-payload-transformer";
+
+export class AvataxOrderFullyRefundedAdapter
+  implements WebhookAdapter<OrderFullyRefundedPayload, void>
+{
+  private logger = createLogger("AvataxOrderFullyRefundedAdapter");
+
+  constructor(private readonly config: AvataxConfig) {}
+
+  async send(payload: OrderFullyRefundedPayload) {
+    this.logger.debug(
+      { payload },
+      "Transforming the Saleor payload for refunding order with AvaTax...",
+    );
+
+    if (!this.config.isAutocommit) {
+      throw new ExpectedError(
+        "Unable to refund transaction. AvaTax can only refund committed transactions.",
+      );
+    }
```

Comment:
Im not sure if this is correct behavior. Correct me if I'm wrong.

As far as I understand, "committing" transaction in avatax is setting some sort of "ready" state. It *can* be done via autocommit, but also can be done [via API](https://developer.avalara.com/api-reference/avatax/rest/v2/methods/Transactions/CommitTransaction/). So, client doesn't have to enable `autocommit` checkbox, they can use other workflows that commit transactions in some cases (eg product is shipped → make invoice). 

If I'm right, this condition should be removed and either:
1. Just try to send it to avatax and if it doesn't work, return error to Saleor 
2. Fetch transaction from Avatax, check if its committed, then react on that